Utilities/bootstrap

```diff
@@ -770,22 +818,17 @@ def add_rpath(rpath, binary):
 #
 # This method removes the hardcoded stdlib path from the binary.
 def installBinary(binary_path, install_path, args, add_rpaths=[], delete_rpaths=[]):
+    note("installing %s: %s" % (os.path.basename(binary_path), ' '.join(cmd)))
     mkdir_p(install_path)
-    cmd = ["rsync", "-a", binary_path, install_path]
-    note("installing %s: %s" % (
-        os.path.basename(binary_path), ' '.join(cmd)))
-    result = subprocess.call(cmd)
-    if result != 0:
-        error("install failed with exit status %d" % (result,))
+    shutil.copy(binary_path, install_path)
```

Comment:
This should be incremental. Maybe use copy on windows?

Reply:
Is there no incremental copying thing in Python? Also, you might be able to use `xcopy` to get incremental copying semantics similar to `rsync` on Windows.

Reply:
Okay, I don't really care if it is incremental or not, I just want a definite answer.  I had made this incremental (I used `distutil` which does allow that).  @ddunbar asked me to remove it and said to just always copy.

Reply:
Sorry, it doesn't _need_ to be incremental but it's important for some of my (internal) workflows. I don't know much about `distutil` but if that's discouraged then it seems fine using rsync where it's available and regular copy as a fallback.

Reply:
Well, `distutils` is part of the python standard distribution, so seems to me that it is better to just use that rather than rsync.